docs/user_guide/emummc/launching_cfw.md

```diff
@@ -2,13 +2,10 @@
 
 Now that the preparation work is out of the way, we're finally ready to launch custom firmware on the Switch.
 
-Unlike systems such as the DSi, Wii, or 3DS, Switch CFW is currently volatile- it will only work as long as your Switch is on. As soon as your Switch completely loses power for any reason (shutting down, battery dying, etc.), CFW will no longer be active and you will need to follow these instructions again.
+Unlike systems such as the DSi, Wii, or 3DS, Switch CFW is currently volatile. It will only work as long as your Switch is on. As soon as your Switch completely loses power for any reason (shutting down, battery dying, etc.), CFW will no longer be active and you will need to follow these instructions again.
 
 &nbsp;
 
-!!!danger "Keep emuMMC offline at all times"
```

Comment:
not sure if i want to remove every mention of keeping the emunand offline from this step of the guide; its good that we mention it earlier but you really should only go online with *one* of your nands, not two. sure you cant put an explanation of that sorts here instead of nuking the entire warning?